addons/analytic/static/src/components/analytic_distribution/analytic_distribution.scss

```diff
@@ -31,23 +60,24 @@
         }
     }
 
-    .tag-list {
-        .badge {
-            margin: 1px 2px 1px 0;
-            border: none;
-
-            .o_delete {
-                padding-left: 4px;
-            }
+    .o_field_analytic_distribution_tags {
+        .o_input_dropdown {
+            flex: 1 0 100px;
         }
     }
 
-    .hidden-focus {
-        // focus to this element is required but we want to hide the blue outline
-        &:focus {
-            // z-index: -10;
-            outline: none;
+    .badge {
+        margin: 1px 2px 1px 0;
+        border: none;
+        font-size: 12px;
```

Comment:
we should not use px values - there ought to be a font size bootstrap variable that defines this.
the one from tags_list.scss seems to apply, so we can remove this one.